Xamarin.Forms.Platform.iOS/Renderers/ShellPageRendererTracker.cs

```diff
@@ -210,16 +213,34 @@ protected virtual async Task UpdateToolbarItems()
 			}
 			else if (IsRootPage && _flyoutBehavior == FlyoutBehavior.Flyout)
 			{
-				ImageSource image = "3bar.png";
-				var icon = await image.GetNativeImageAsync();
-				NavigationItem.LeftBarButtonItem = new UIBarButtonItem(icon, UIBarButtonItemStyle.Plain, OnMenuButtonPressed);
+
+				await SetDrawerArrowDrawableFromFlyoutIcon();
 			}
 			else
 			{
 				NavigationItem.LeftBarButtonItem = null;
 			}
 		}
 
+		async Task SetDrawerArrowDrawableFromFlyoutIcon()
+		{
+			Element item = Page;
+			ImageSource image = null;
+			while (!Application.IsApplicationOrNull(item))
+			{
+				if (item is IShellController shell)
+				{
+					image = shell.FlyoutIcon;
+					item = null;
+				}
+				item = item?.Parent;
+			}
+			if (image == null)
+				image = "3bar.png";
```

Comment:
Perhaps `image = ImageSource.FromResource("3bar.png");`

Reply:
we need something better, leaving for now, but we might want to draw the 3 bars

Reply:
let's track this when e try fix for good #3124